components/clp-package-utils/clp_package_utils/scripts/native/decompress.py

```diff
@@ -25,15 +26,64 @@
 logger.addHandler(logging_console_handler)
 
 
-def decompress_paths(
+def _validate_and_load_config_file(
```

Comment:
can't call it `validate_and_load_config_file` because the function in `general.py` already has this name.

Reply:
How about renaming the existing `validate_and_load_config_file` to `load_config_file`?